db/migrate/20230601025347_create_search_results.rb

```diff
@@ -0,0 +1,10 @@
+class CreateSearchResults < ActiveRecord::Migration[7.0]
+  def change
+    create_table :search_results do |t|
+      t.string :keyword
+      t.text :html_code
```

Comment:
In the AC, I saw an `Update the status accordingly` requirement, so should we add it to this PR as well? 🤔

Reply:
My bad, I think it would be better to keep it as part of this issue: #19 🙏

Reply:
@hoangmirs As discussed with Toby, the status should be updated accordingly when the search job starts and the scraping data is completed 🙏